src/states/bootstrapping.rs

```diff
@@ -303,9 +307,15 @@ impl Bootstrapping {
     fn handle_direct_message(
         &mut self,
         direct_message: DirectMessage,
+        signature: sign::Signature,
         pub_info: PublicInfo,
     ) -> Transition {
         use self::DirectMessage::*;
+
+        if !self.verify_direct_message(&direct_message, &signature, &pub_info) {
+            return Transition::Stay;
```

Comment:
Do we need to carry out actions on this verification failure?
maybe worth to put a comment here explain why not or why required.
and same to couple of other places carrying out this new verification check.

Reply:
No, I don't think so - I think the code is self-documenting enough.  We don't need to do anything other than log the failure (which we do inside `verify_direct_message()`).  What would you suggest as a comment?

Reply:
ok, if we don't need to carry out any other action on this failure, then we don't need extra comment.